### ISSUE TITLE
Regression for Custom Tag validation

### ISSUE THREAD

**Issue by keesschollaart81** [redhat-developer/yaml-language-server#175]
It seems that the 5.x version of this package introduced a bug with validation for custom tags. Using version 4.x, validation for custom tags seems to be working fine using this YAML code:

```YAML
hello:
  world: from
  Rotterdam: !customTag tagValue
```

With version 5.x (when the `customTag` is not registered) an error is returned, pointing to the wrong line/character position of `!customTag`.

I've tested it by updating [this test](https://github.com/redhat-developer/yaml-language-server/blob/master/test/customTags.test.ts#L68-L80) to:

```TS
describe('Test that validation does throw errors', function () {
    it('Error when custom tag is not available', done => {
        const content = 'hello:\n  world: from\n  Rotterdam: !customTag tagValue';
        const validator = parseSetup(content, []);
        validator.then(function (result) {
            assert.equal(result.length, 1);
            assert.deepEqual(
                result[0],
                createExpectedError('unknown tag <!customTag>', 2, 13, 2, 26)
            );
        }).then(done, done);
    });
});
```

Then the test above fails because the error is pointing to [line 1 character 46](https://prnt.sc/otwnvu) both for start and end compared to line 2 character 13 in 4.x. [This](https://prnt.sc/otwt2y) is what it looked like before with the same YAML.

I develop the [VS Code Extension for Home Assistant](https://github.com/keesschollaart81/vscode-home-assistant/) and I'm trying to migrate to 5.x but this bug causes wrong errors like this:

![image](https://user-images.githubusercontent.com/6755359/63213888-ddb8ee80-c111-11e9-8545-ba20a97c1ee1.png)

I'm not sure how to fix this, hopefully the failing test ^ helps to fix it?

**Comment by JPinkney** [comment]
Yeah this is a weird error. AFAIK nothing major was changed with custom tags for 5.x. I can reproduce though so I'll take a look when I get a chance!

**Comment by JPinkney** [comment]
This is now fixed in 0.9.1
